js/src/components/paid-ads/ads-campaign.js

```diff
@@ -45,15 +60,24 @@ export default function AdsCampaign( { onContinue } ) {
 					{
 						link: (
 							<AppDocumentationLink
-								context="create-ads"
+								context={
+									isCreation ? 'create-ads' : 'edit-ads'
+								}
 								linkId="see-what-ads-look-like"
 								href="https://support.google.com/google-ads/answer/6275294"
 							/>
 						),
 					}
 				) }
 			/>
-			<CreateCampaignFormContent formProps={ formContext } />
+			{ isCreation ? (
+				<CreateCampaignFormContent formProps={ formContext } />
+			) : (
+				<EditPaidAdsCampaignFormContent
+					formProps={ formContext }
+					allowMultiple={ campaign.allowMultiple }
+				/>
```

Comment:
💅  This is what I meant in our previous discussion in https://github.com/woocommerce/google-listings-and-ads/pull/1794#discussion_r1033216007 here is clearer the divergence in naming:

`CreateCampaignFormContent ` and `EditPaidAdsCampaignFormContent` seems not consistent in naming.

I'd suggest this 2 options:

1. `CreateCampaignFormContent` & `EditCampaignFormContent`
2. `CreatePaidAdsCampaignFormContent` & `EditPaidAdsCampaignFormContent`

Reply:
Sorry, I misunderstood you were asking to rename the `AdsCampaign` to `AdsCampaignForm` or `AdsCampaignFormContent` in https://github.com/woocommerce/google-listings-and-ads/pull/1794#discussion_r1033216007.

Since `CreateCampaignFormContent` and `EditPaidAdsCampaignFormContent` were not implemented in this PR or this project, they are a little out of scope. But it may be a good opportunity to handle some of the name ambiguities and code duplication further.

I opened a separate PR #1803 to:

- Handle inconsistent component names
- Eliminate the ambiguous component hierarchy caused by [my previous reference to *FormContent names](https://github.com/woocommerce/google-listings-and-ads/pull/1794#discussion_r1035576050)